test/etherscan.js

```diff
@@ -31,6 +31,10 @@ chai.use(chaiHttp);
 const CUSTOM_PORT = 5678;
 
 describe("Import From Etherscan and Verify", function () {
+  this.beforeEach(async function () {
```

Comment:
I added this, because lots of Etherscan tests were failing

Reply:
I don't think this is a reason for the tests to fail. The rate limit is quite generous. Did you look why it was failing?

Reply:
If some etherscan tests fails then they end sooner than expected, hitting Etherscan's API rate limit